src/components/BottomModal.js

```diff
@@ -50,14 +51,14 @@ const BottomModal = ({
   return (
     <ButtonUseTouchable.Provider value={true}>
       <ReactNativeModal
+        {...rest}
+        {...backDropProps}
         isVisible={isOpened}
         deviceWidth={width}
         deviceHeight={height}
         useNativeDriver
         hideModalContentWhileAnimating
         style={styles.root}
-        {...backDropProps}
-        {...rest}
```

Comment:
did you check we never need to override the props like `style` ?

Reply:
no i did not.... will fix that